src/components/20-molecules/accordion/test/basic/e2e.js

```diff
@@ -53,50 +53,8 @@ test.describe('accordion', () => {
 
     expect(
       await page
-        .locator('.m-accordion__title-container--disabled >> nth=0')
+        .locator('.m-accordion__button >> nth=0')
```

Comment:
Had to find https://playwright.dev/docs/selectors#n-th-element-selector to understand this, might be worth a comment pointing there? Not sure.